docs/howto/custom-rootfs-multiple-targets.md

```diff
@@ -0,0 +1,314 @@
+export COMPUTER_NAME=Carbonizer
+
+# Path to the cloud-config file whose contents will be used to initialize the WSL instances:
+export CLOUD_INIT_FILE="~/Downloads/init.yaml"
+```
+
+````
+
+````{group-tab} PowerShell
+```powershell
+# Credentials to authenticate the API requests
+$LANDSCAPE_USER_EMAIL="admin@mib.com"
+$LANDSCAPE_USER_PASSWORD="mib"
+$LANDSCAPE_URL="https://landscape.mib.com"
+
+# The URL of the custom rootfs to be deployed:
+$ROOTFS_URL="http://landscape.mib.com:9009/ubuntu-24.04-custom.tar.gz"
+
+# The list of IDs of the different Windows machines on which we are going to deploy WSL instances:
+$PARENT_COMPUTER_IDS=@(26, 30, 31)
+
+# The name of the WSL instance to be created:
+$COMPUTER_NAME="Carbonizer"
+
+# Path to the cloud-config file whose contents will be used to initialize the WSL instances:
+$CLOUD_INIT_FILE="~/Downloads/init.yaml"
+```
+
+````
+
+`````
+
+
+Generate a Base64-encoded string with the cloud-config data:
+
+`````{tabs}
+
+````{group-tab} Bash
+
+```bash
+BASE64_ENCODED_CLOUD_INIT=$(cat $CLOUD_INIT_FILE | base64 --wrap=0)
+```
+
+````
+
+````{group-tab} PowerShell
+```powershell
+$content = Get-Content -Path $CLOUD_INIT_FILE -Raw
+$bytes = [System.Text.Encoding]::UTF8.GetBytes($content)
+$BASE64_ENCODED_CLOUD_INIT = [System.Convert]::ToBase64String($bytes)
+```
+
+````
+
+`````
+
+## Authenticate against the Landscape API
+
+Build the authentication payload of the form: `{"email": "admin@mib.com", "password": "mib"}` using the values exported in prior steps:
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+LOGIN_JSON=$( jq -n \
+    --arg em "$LANDSCAPE_USER_EMAIL" \
+    --arg pwd "$LANDSCAPE_USER_PASSWORD" \
+    '{email: $em, password: $pwd}' )
+```
+````
+
+````{group-tab} PowerShell
+```powershell
+$LOGIN_JSON = @{
+ email = "$LANDSCAPE_USER_EMAIL"
+ password = "$LANDSCAPE_USER_PASSWORD"
+} | ConvertTo-Json
+```
+````
+`````
+
+Issue an authenticate request and retrieve the JWT to be used in the subsequent API requests.
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+LOGIN_RESPONSE=$( curl -s -X POST "$LANDSCAPE_URL/api/v2/login" \
+    --data "$LOGIN_JSON"                                        \
+    --header "Content-Type: application/json"                   \
+    --header "Accept: application/json" )
+
+JWT=$( echo $LOGIN_RESPONSE | jq .token | tr -d '"')
+```
+````
+
+````{group-tab} PowerShell
+```powershell
+$LOGIN_RESPONSE = Invoke-WebRequest -Method POST `
+    -URI "$LANDSCAPE_URL/api/v2/login"           `
+    -Body "$LOGIN_JSON" -ContentType "application/json"
+
+$JWT = ConvertTo-SecureString -AsPlainText -Force $( $LOGIN_RESPONSE.Content | ConvertFrom-Json).token
+```
+````
+
+`````
+
+## Send the Install request
+
+Build the payload with information about the WSL instance to be deployed. In this case it would look like
+`{"rootfs_url": "http://landscape.mib.com:9009/ubuntu-24.04-custom.tar.gz", "computer_name": "Carbonizer", "cloud_init": <base64 encoded material>}`.
```

Comment:
The in-line JSON is a little hard to read.
You could put it on a separate line or even in a (prettified) code block.